src/cardano/CardanoEngine.ts

```diff
@@ -521,6 +519,7 @@ export class CardanoEngine extends CurrencyEngine<
     vkeyWitnesses.add(vkeyWitness)
 
     if (isStakeTx) {
+      const stakeKey = accountKey.derive(2).derive(0)
```

Comment:
This belongs in "Add isStakeTx otherParams field for signing with stake key" commit